app/controllers/concerns/test_track/controller.rb

```diff
@@ -2,6 +2,8 @@ module TestTrack::Controller
   extend ActiveSupport::Concern
 
   included do
+    class_attribute :test_track_identity
```

Comment:
I decided to use a class_attribute for this because it does inheritance correctly out of the box regardless of the crazy controller hierarchies people come up with.